### PR TITLE
Add documentation for v1.4.2 benchmarks.

### PR DESCRIPTION
**Pull request by carolili**
* Creating a subfolder for benchmarks in the documentation.
* Keeping the same page layout as for the previous benchmarks.
* Minor additions to the conclusions of the previous benchmarks.
* Including results for benchmarks with authentication enabled.

### CODE REVIEW

**Review by imsdu — CHANGES_REQUESTED** [review]

File: docs/src/main/paradox/docs/delta/benchmarks/v1.4.2.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+*   [AWS EBS storage](https://aws.amazon.com/ebs/) - gp3, 10 000 IOPS, throughput 250 MB/s
+*   Nginx ingress controller
+*   1 Keycloak pod using its internal database H2 as storage, 5GB gp2 storage
+*   1 to 12 Cassandra pods each with 3.5 vCPU, 12 GB HEAP, 250 GB EBS storage
+*   1 Elasticsearch pod with 2 vCPU, 8 GB HEAP, 50GB gp2 storage
+*   1 Blazegraph pod with 2 vCPU, 4 GB HEAP, 50GB gp2 storage
+*   1 Prometheus pod and 1 Grafana pod, 50GB gp2 storage
+
+The services were deployed as follows:
+
```
Comment (bogdanromanx) [inline docs/src/main/paradox/docs/delta/benchmarks/v1.4.2.md:21]: I would remove these lines, add the delta config next to the others and order them as follows:
- delta
- cassandra
- elastic
- bg
- keycloak
- prometheus
- nginx

**Review by bogdanromanx — CHANGES_REQUESTED** [review]

**Review by bogdanromanx — CHANGES_REQUESTED** [review]
I would recommend exporting the charts in PNG format for better browser compatibility.

File: docs/src/main/paradox/docs/delta/benchmarks/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+*   **throughput** - how many requests per second the system can handle
+*   **latency** - the time the system needed to provide response to the requests
+
+... and how they were affected by different factors, especially:
+
+*   **data volume** - how does the volume of the data in the system affect the performance
+*   **hardware configuration and scalability** - does assigning more hardware increase the performance of the system and
+    can the system scale both horizontally and vertically.
+*   **clustering** - what's the effect of changing from a single node to clustered deployment, as well as, what's the
```
[the remark this answers is not in the document]
Comment (carolili) [reply docs/src/main/paradox/docs/delta/benchmarks/index.md]: Thanks, fixed.

File: docs/src/main/paradox/docs/delta/benchmarks/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    effect of adding more nodes to the cluster.
+
+The test scenarios and scripts can be found in the @link:[nexus-benchmarks](https://github.com/BlueBrain/nexus-benchmarks){ open=new }
+repository.
+
+The latest benchmarks are the Nexus Delta v1.4.2 @ref:[benchmarks](v1.4.2.md).
```
[the remark this answers is not in the document]
Comment (carolili) [reply docs/src/main/paradox/docs/delta/benchmarks/index.md]: Thanks, fixed.

File: docs/src/main/paradox/docs/delta/benchmarks/v1.4.2.md (hunk 1 above)
Comment (carolili) [reply docs/src/main/paradox/docs/delta/benchmarks/v1.4.2.md:15]: Change done and charts exported in PNG format.

**Review by bogdanromanx — APPROVED** [review]

**Pull request merged**